org.manasource.Mana.json

```diff
@@ -0,0 +1,33 @@
+{
+  "app-id": "org.manasource.Mana",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "mana",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--persist=.config/mana",
+    "--persist=.local/share/mana"
+  ],
+  "modules": [
+    "SDL2_gfx.json",
+    "Guichan.json",
```

Comment:
this should be inline in the manifest.

Reply:
I thought this way it could be cleaner and more useful for others, but sure I will put them inline.